relay-cardinality/src/redis/limiter.rs

```diff
@@ -58,12 +56,12 @@ impl RedisSetLimiter {
     fn check_limits(
         &self,
         con: &mut Connection,
-        state: &mut LimitState<'_>,
+        state: &LimitState<'_>,
+        scope: QuotaScoping,
```

Comment:
`LimitState` also contains a quota scoping too. Could we clarify we should use the one in the function's argument?

Ideally there'd be one, but I don't have suggestions on how to accomplish that.

Reply:
`LimitState` contains the partial scope not the full scope.